modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -15,5 +15,7 @@ In addition, Sensor is responsible for all cluster interactions, such as applyin
 * *Admission controller*: The Admission controller prevents users from creating workloads that violate security policies in {product-title-managed-short}.
 * *Collector*: Collector analyzes and monitors container activity on cluster nodes.
 It collects container runtime and network activity information and sends the collected data to Sensor.
-* *Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
+* *Scanner-slim*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
 However, on {ocp}, {product-title-managed-short} installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
+* *Scanner V4 Indexer*: This component is optional in release 4.4. Scanner V4 provides additional scanning features, such as consistent and accurate vulnerability scan results across the Red Hat product ecosystem; expanded language vulnerability scanning support; expanded operating system scanning support; and an additional vulnerability database source the Open Source Vulnerabilities (OSV) database.
```

Comment:
> and an additional vulnerability database source the Open Source Vulnerabilities (OSV) database.

Indexer does not provide this

Reply:
@mclasmeier can you remind me, if we deploy Scanner V4 in a Secured Cluster, will we also get a Scanner-slim or no? We don't need it, but I'm not sure if we still deploy it

Reply:
@dcaravel, I think there's a misunderstanding here -- sorry that I have overlooked this one until now.

The smooth upgrade experience discussion took place before we switched from opt-out to opt-in. Furthermore, I think it was the other way around: If we had `scanner.disable: false` on secured-cluster side then we would also set `scannerV4.disable: false` in case it was not explicitly set by the user. But, as I said, this logic was removed with the switch to opt-out.

So, the logic for secured-clusters at the moment is this: you *need* `scanner.disable: false` to enable the local scanning functionality, i.e. `LOCAL_IMAGE_SCANNING_ENABLED=true` for sensor (let me ignore the edge-case of deploying secured-cluster and central into the same namespace for now, IIRC it works slightly different there).  And *in addition* to that you can enable scannerV4. Enabling Scanner V4 without enabling Scanner V2 won't enable local image scanning, but it will deploy scanner V4 components.

It has been my understanding so far that this is a pretty much untested and unsupported scenario?

Reply:
Thanks @mclasmeier for the clarification. Rephrasing:

If want to use Scanner V4 in the secured cluster, both scanner-slim and V4 need to be installed by setting both `scanner.disable: false` and `scannerV4.disable: false`.

If only `scannerV4.disable: false` is set, V4 will be installed however `ROX_LOCAL_IMAGE_SCANNING_ENABLED` will not be set on Sensor and therefore no scanning will be done (unsupported setup).

If only `scanner.disable: false` is set, then only Scanner-Slim is installed.